images/scripts/get-image-digest.sh

```diff
@@ -12,4 +12,10 @@ if [ "$#" -ne 1 ] ; then
   exit 1
 fi
 
-docker buildx imagetools inspect "${1}" 2>/dev/null | grep Digest | awk '{ print $2 }' || true
+inspect=$(docker buildx imagetools inspect "${1}" --raw 2>/dev/null | sha256sum | cut -d " " -f 1 )
```

Comment:
This is not meant to run on the developers machine ever, right? From what I remember, Darwin's hash functions are invoked differently than Linux

Reply:
oh I didn't know that... what output do you get on mac os for let's say this:
```
$ ./images/scripts/get-image-digest.sh docker.io/cilium/cilium:v1.7.14
sha256:0bd894e9dc0188482bc0316c2889a0aa9b2fce3a3c9e4dc6f55afb4a2afbf84b
```

Reply:
I don't have Mac OS, but I remember this from when I used to use it in a past life :wink:

Reply:
> ```
> $ ./images/scripts/get-image-digest.sh docker.io/cilium/cilium:v1.7.14
> sha256:0bd894e9dc0188482bc0316c2889a0aa9b2fce3a3c9e4dc6f55afb4a2afbf84b
> ```

@aanm It doesn't print anything.

Reply:
@aditighag can you try again please?

Reply:
shrug, openssl doesn't exist in the GH action... nevermind...

Reply:
As long as the script is intended for GH Actions, then maybe we don't need worry about this

Reply:
> @aditighag can you try again please?

@aanm The command [1] itself works, but the script doesn't print anything. I narrowed it down to `set -o errexit`. I'll need to look into this further. 

```
$ docker buildx imagetools inspect "${1}" --raw 2>/dev/null | openssl sha256 -r | cut -d " " -f 1
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
$ ./images/scripts/get-image-digest.sh docker.io/cilium/cilium:v1.7.14
```

[1] `docker buildx imagetools inspect "${1}" --raw 2>/dev/null | openssl sha256 -r | cut -d " " -f 1`